<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSettings</class>
 <widget class="QDialog" name="UserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>831</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>43</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,5,5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Operator Name : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="iconSize">
           <size>
            <width>46</width>
            <height>46</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Dashboard Colour : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Transmission Type : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="AddOperator_Button">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add Operator</string>
        </property>
        <property name="iconSize">
         <size>
          <width>69</width>
          <height>63</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Load_Button">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Update_button">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
